src/client.js

```diff
@@ -33,21 +33,22 @@ var events = [
 	"whois"
 ];
 var inputs = [
+	// These inputs are sorted in order that is most likely to be used
+	"msg",
+	"whois",
+	"part",
 	"action",
+
 	"connect",
 	"invite",
 	"join",
 	"kick",
 	"mode",
-	"msg",
 	"nick",
 	"notice",
-	"part",
 	"quit",
 	"raw",
-	"services",
 	"topic",
-	"whois"
```

Comment:
After getting that the point of this PR was to avoid a full rewrite, you can disregard this earlier comment.
I'll implement this in a further PR as well as include a bunch of tests for these at the same time, I should get to it within the next days.

By the way, I agree with this order, was just saying it could get opinionated at some point, but that won't matter when I add direct mapping.